images/linux/scripts/tests/Haskell.Tests.ps1

```diff
@@ -32,6 +33,10 @@ Describe "Haskell" {
         "stack --version" | Should -ReturnZeroExitCode
     }
 
+    It "GHCup dir has correct user permissions" {
+        "stat --format '%U' $GHCupCommonPath" | Should -Be "${env:USER}"
```

Comment:
I'm not sure if this is correct powershell and will work... but this is the idea at least

Reply:
variable name is `GHCCommonPath`

Reply:
No, this is a new variable

Reply:
@hasufell test is failing because of $USER in the moment of changing permissions and performing the test is not the same as the <actionuser>. action user is 'runner'

Reply:
So what is the right way to invoke test as if you're executing a github action?

Reply:
When changing owner permissions we should do:
`chown -R "runner" $GHCUP_INSTALL_BASE_PREFIX/.ghcup`
And for the test:
`"stat --format '%U' $GHCupCommonPath" | Should -Be "runner"`

Reply:
Can you add a commit fixing that? I'm unsure when I'll be at my PC.

Reply:
Fixed